nautobot/extras/models/groups.py

```diff
@@ -91,52 +259,44 @@ def count(self):
         """Return the number of objects in the group."""
         return self.get_queryset().count()
 
-    @property
-    def map(self):
-        """
-        Accessor to automatically generated `BaseDynamicGroupMap` class.
+    def get_absolute_url(self):
+        return reverse("extras:dynamicgroup", kwargs={"slug": self.slug})
 
-        This class object is cached on the instance after the first time it is accessed.
-        """
+    @property
+    def base_url(self):
+        if self.model is None:
+            return ""
 
-        if getattr(self, "_map", None) is None:
-            try:
-                model = self.content_type.model_class()
-                dynamicgroupmap_class = dynamicgroup_map_factory(model)
-            except (models.ObjectDoesNotExist, TypeError):
-                dynamicgroupmap_class = None
+        model = self.content_type.model_class()
+        route_name = get_route_for_model(model, "list")
+        return reverse(route_name)
 
-            self._map = dynamicgroupmap_class
+    def urlencode(self, filter_params):
```

Comment:
Also wondering if we really need this as an API method, versus just directly calling `urllib.parse.urlencode` as needed?

Reply:
Yeah this is a leave over from the group map. Thanks.